src/suggest.rs

```diff
@@ -1,18 +1,9 @@
 use std::collections::{HashMap, HashSet};
 
-use serde::Deserialize;
-
 use crate::{trie::Trie, utils::fix_string};
 
-#[derive(Deserialize)]
-#[serde(rename_all = "camelCase")]
-pub struct Block {
-    pub transliterate: Vec<String>,
-    pub entire_block_optional: Option<bool>,
-}
-
 pub struct Suggest {
-    patterns: HashMap<String, Block>,
+    patterns: HashMap<String, Vec<String>>,
```

Comment:
Can you add a comment mentioning that if the array contains an empty string `""`, the whole pattern would be considered as optional?

Reply:
@gulshan